game/settings.py

```diff
@@ -45,6 +53,7 @@ class Settings:
     perf_moving_units: bool = True
     perf_infantry: bool = True
     perf_destroyed_units: bool = True
+    maximum_ground_units_in_base: int = 50
```

Comment:
It's only a max for the auto-procurer. `ground_unit_procurement_target_per_base`?

Also, https://github.com/dcs-liberation/dcs_liberation/issues/316 is yours now :) I'd assign it but it looks like you haven't accepted the project invite yet?